### PR TITLE
Test ruby 2.7/3.0

### PR DESCRIPTION
**Pull request by jrafanie**
Test ruby 2.7/3.0

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
+        ruby-version: "${{ matrix.ruby-version }}"
         bundler-cache: true
       timeout-minutes: 30
     - name: Prepare tests
       run: bin/setup
     - name: Run tests
       run: bundle exec rake
     - name: Report code coverage
-      if: ${{ github.ref == 'refs/heads/master' && matrix.ruby-version == '2.7' }}
+      if: "${{ github.ref == 'refs/heads/master' && matrix.ruby-version == '2.7' }}"
```
Comment (jrafanie) [inline .github/workflows/ci.yaml:47]: We'll update this separately

Comment (kbrock) [reply .github/workflows/ci.yaml:47]: what did you just say?

Comment (jrafanie) [reply .github/workflows/ci.yaml:47]: We'll update all the code coverage lines that run with 2.7 to use 3.0 later on.

**Comment by jrafanie** [comment]
YOLO merge

**Pull request merged**